### PR TITLE
[DRAFT]vdk-jupyter: ci/cd

### PR DESCRIPTION
**Pull request by duyguHsnHsn**

### CODE REVIEW

File: projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  retry_options:
+    max: 1
+    when:
+      - always
+
+
+build-vdk-jupyterlab-extension:
+  stage: build
+  extends: .build-plugin-dind
+  image: "python:3.8-alpine"
```
Comment (ivakoleva) [inline projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml:14]: what image do the rest of the VDK gitlab ci jobs leverage for running python workloads?

File: projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
```
Comment (ivakoleva) [inline projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml:8]: same snippet duplicates .plugin-common.yml

Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml:8]: this is work in progress I will be refactoring the script a lot

Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml:8]: @ivakoleva currently the installation is failing on random occasions and I am using this pr to find some pattern

File: projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml (hunk 1 above)
Comment (duyguHsnHsn) [reply projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml:14]: @ivakoleva I used that image to use apk for adding npm

**Pull request closed without merging**